app/src/main/java/space/taran/arknavigator/mvp/model/repo/kind/ResourceKindFactory.kt

```diff
@@ -3,12 +3,15 @@ package space.taran.arknavigator.mvp.model.repo.kind
 import space.taran.arknavigator.mvp.model.dao.ResourceExtra
 import space.taran.arknavigator.mvp.model.repo.index.ResourceId
 import space.taran.arknavigator.utils.extension
+import space.taran.arknavigator.utils.getMimeTypeUsingTika
 import java.nio.file.Path
 
 interface ResourceKindFactory<T : ResourceKind> {
     val acceptedExtensions: Set<String>
+    val acceptedMimeTypes: Set<String>
     val acceptedKindCode: KindCode
-    fun isValid(path: Path) = acceptedExtensions.contains(extension(path))
+    fun isValid(path: Path) = acceptedExtensions.contains(extension(path)) ||
+        acceptedMimeTypes.contains(getMimeTypeUsingTika(path))
```

Comment:
Maybe we should log warning if MIME type conflicts with extension.

Reply:
Currently we get conflict for svg file without extension.  So please tell me the case of the conflict for others ?

Reply:
We can check it in generic way, something like this:
```
val extensionIsValid = acceptedExtensions.contains(extension(path))
val mimeTypeIsValid = acceptedMimeType.contains(getMimeTypeUsingTika(path))
if (extensionIsValid != mimeTypeIsValid) {
    log.warn("File extension is valid? ${extensionIsValid}")
    log.warn("MIME type is valid? ${mimeTypeIsValid}")
}
```